content/view/view.html

```diff
@@ -12,7 +12,7 @@ <h4 id="publicProfileUsername">{{username}}</h4>
 		<hr>
 	</div>
 	<div ng-show="!linkname && !error" class="do-not-animate">
-			<h4 id="publicProfileUsername">Retrieving Profile...</h4>
+			<img id="spinner" ng-src="image/spinner.gif" class="fadeFast">
```

Comment:
Won't show the spinner properly here but I will make a PR to fix it after you've merged this one.